src/main/java/Ui.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * This class handles the User Interface of Duke.
+ *
+ * @author richwill28
+ */
+class Ui {
+    /** Decorative logo */
+    public static final String LOGO =

```

Comment:
As mentioned, it might be worth considering removing comments like these, especially so when the names you have chosen for such class members seem to clearly suggest what they refer to.

Reply:
Thanks for the suggestion! I agree that this comment introduce unnecessary clutter on my code, will remove it on the next commit.